docs/discovery-handlers/opc-ua.md

```diff
@@ -1,80 +1,87 @@
-# Configuring Akri to Discover Devices via OPC UA
+# OPC UA
+
 ## Background
-OPC UA (Object Linking and Embedding for Process Control Unified Architecture) is a communication protocol for
-industrial automation. Akri has implemented a Discovery Handler for discovering OPC UA Servers that live at specified endpoints or are registered with specified Local Discovery Servers. Background on the OPC UA Discovery Handler implementation can be found in the [proposal](proposals/opcua.md). To try out using Akri to discover and utilize OPC UA servers, see the [OPC UA end-to-end demo](./opcua-demo.md).
 
-All of Akri's components can be deployed by specifying values in its Helm chart during an installation. This section will cover the values that should be set to (1) deploy the OPC UA Discovery Handlers and (2) apply a Configuration that tells Akri to discover devices using that Discovery Handler.
+OPC UA \(Object Linking and Embedding for Process Control Unified Architecture\) is a communication protocol for industrial automation. Akri has implemented a Discovery Handler for discovering OPC UA Servers that live at specified endpoints or are registered with specified Local Discovery Servers. Background on the OPC UA Discovery Handler implementation can be found in the [proposal](proposals/opcua.md). To try out using Akri to discover and utilize OPC UA servers, see the [OPC UA end-to-end demo](opcua-demo.md).
```

Comment:
`[proposal]`  link in OPC-UA directs to an non-existing page `https://proposal/opcua.md`. I assume it should be:

    [proposal](../proposals/opcua.md)
    
 similar Linking problems are evident through out the OPC-UA and udev pages.

Reply:
With Gitbook, there's no proposal folder anymore, just a section in the side. I'll update the instructions to reflect this.